var/spack/repos/builtin/packages/r-lobstr/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RLobstr(RPackage):
+    """lobstr provides tools in the same vein as str(), which allow you to dig into
+    the detail of an object"""
```

Comment:
Could you use the title and description from the `DESCRIPTION` file, also on the CRAN web site, so that the Spack package stays consistent with CRAN?